### ISSUE TITLE
Incorrect boolean to string conversion

### ISSUE THREAD

**Issue by vmartynets** [sendinblue/APIv3-csharp-library#11]
Hi!
I'm tried to use this SDK to fetch email template as following:
```cs
var templates = apiInstance.GetSmtpTemplates(true);
```
but received a weird error:
```
sib_api_v3_sdk.Client.ApiException: 'Error calling GetSmtpTemplates: {"code":"invalid_parameter","message":"Template status should be either true or false"}'
```

There is a bug in [`ApiClient.ParameterToString`](https://github.com/sendinblue/APIv3-csharp-library/blob/master/src/sib_api_v3_sdk/Client/ApiClient.cs#L247) method when it tries converting boolean to string it returns boolean as string but with capital letter. Quick example:
![image](https://user-images.githubusercontent.com/1114365/79448759-ad03be80-7fea-11ea-9401-47428004e9eb.png)

Possible fix - add following branching to `ParameterToString` method:
```cs
 else if (obj is bool)
    return ((bool)obj).ToString().ToLower();
```

**Comment by umair-me** [comment]
Fixed in #12

**Comment by amitsendinblue** [comment]
mentioned PR incorporated in new release - https://github.com/sendinblue/APIv3-csharp-library/releases/tag/v2.1.11
